src/libraries/System.Net.Sockets/tests/FunctionalTests/TcpClientTest.cs

```diff
@@ -168,6 +169,10 @@ public async Task ConnectAsync_DnsEndPoint_Success(int mode)
                         await client.ConnectAsync(addresses[0], port, CancellationToken.None);
                         break;
                     case 8:
+                        addresses = await Dns.GetHostAddressesAsync(host);
+                        await client.ConnectAsync(new IPEndPoint(addresses[0], port), CancellationToken.None);
+                        break;
```

Comment:
Can we add a test for the other overload being added, too?

Reply:
I added a test. Not sure whether those CI errors are caused by the test though. Kind of seems unrelated.